editor/src/document/document_message_handler.rs

```diff
@@ -151,7 +151,7 @@ impl DocumentMessageHandler {
 			};
 
 			let (path, closed) = match &layer.ok()?.data {
-				LayerDataType::Shape(shape) => Some((shape.path.clone(), shape.closed)),
+				LayerDataType::Shape(shape) => Some((shape.path.clone(), shape.path.elements().last() == Some(&kurbo::PathEl::ClosePath))),
```

Comment:
I thought the goal of removing `.closed` was to treat it as having a filled color, not something related to the Kurbo geometry?

Reply:
The `closed` field was used incorrectly here. Probably due to the confusing naming. Here `closed` actually means whether the outline is closed. So the name fits.

Reply:
Gotcha. Checking with @otdavies: does this cause any conflicts with your branch? And is this robust enough when multiple sub-shapes (separated by `moveto`s) are involved?

Reply:
I will remove my usage of .closed and thus won't be an issue. I didn't use it before and I only recently incorporated it.

Reply:
@pkupper: @otdavies and I discussed this and it sounds like the best course of action here is to put a comment above this line: `// TODO: This ClosePath check does not handle all cases, fix this soon` and then we can go ahead and merge your PR right away.